control_plane/roles/control_plane_ib/tasks/infiniband_container_image.yml

```diff
@@ -29,8 +29,8 @@
 - name: Update omnia project path in k8s_infiniband.yml
   replace:
     path: "{{ role_path }}/files/k8s_infiniband.yml"
-    regexp: 'path: /root/omnia'
-    replace: "path: {{ role_path.split('control_plane')[0] }}"
+    regexp: '        - name: omnia-storage\n          hostPath:\n            path:.*'
```

Comment:
@blesson-james, same question. If the play is always executed, then we should just alter the file directly.

Reply:
@lwilson same reason as above